README.md

```diff
@@ -23,6 +23,11 @@ Visualization tools in haystack-ui include -
 | Real time dependency graph with health and connectivity indicators |
 |     ![Service Graph](./public/images/assets/service_graph.png)     |
 
+|                               Service Insights (beta)                                |
+| :----------------------------------------------------------------------------------: |
+| Graph view of all upstream and downstream dependencies related to a selected service |
```

Comment:
Let's remove "Graph" to avoid confusion, how about simply "View of…"?

Also I'd say "dependencies of a selected service" so there isn't wondering what "related to" means.